app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection =sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+# CREATE - C
+    def create(self, first_name: str, last_name:str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+
+# RETRIEVE - R
+    def all(self) -> list:
+        actors.cursor = self._connection.execute(
```

Comment:
There is a typo in the variable name `actors.cursor`. It should be `actors_cursor` to match the variable used in the list comprehension on line 26.